### DISCUSSION TITLE
d3-dsv seems missing

### DISCUSSION THREAD

**Discussion by tvercaut** [cdnjs/cdnjs#14250] [General]
I am interested in using d3-dsv. Unfortunately, while listed, it seems to be missing on cdnjs with no links provided in the corresponding page:
https://cdnjs.com/libraries/d3-dsv

In the main search, the following link and script tag is available to copy, however it leads to a 404:
https://cdnjs.cloudflare.com/ajax/libs/d3-dsv/3.0.0/d3-dsv.min.js
```
<script src="https://cdnjs.cloudflare.com/ajax/libs/d3-dsv/3.0.0/d3-dsv.min.js" referrerpolicy="no-referrer"></script>
```

Finally, I saw this past issue https://github.com/cdnjs/cdnjs/issues/7904 which seems to indicate that d3-dsv should be available.

Should I file this as an issue here or am I missing something?

**Comment by MattIPv4** [comment]
Please open an issue (or PR) in cdnjs/packages, noting what files should be pulled in from the package source. https://cdnjs.com/libraries/d3-dsv/1.0.8 looks to be the last version with files, so I suspect the package structure has changed beyond that point and the auto-update config now isn't matching anything.

**Reply by tvercaut** [reply]
Thanks. The issue is here https://github.com/cdnjs/packages/issues/1802
